README.md

```diff
@@ -1,24 +1,42 @@
-# README
-
-This README would normally document whatever steps are necessary to get the
-application up and running.
-
-Things you may want to cover:
-
-* Ruby version
-
-* System dependencies
-
-* Configuration
-
-* Database creation
-
-* Database initialization
-
-* How to run the test suite
-
-* Services (job queues, cache servers, search engines, etc.)
-
-* Deployment instructions
-
-* ...
+# Introduction
+
+This is a Rails application which demonstrates a system for allowing claimants to initiate and manage unemployment claims, and for customer service representatives to process claims.
+
+# Prerequisites
+
+* [rvm](https://rvm.io/)
+* Ruby 2.7.1 (`rvm install "ruby-2.7.1"`)
+* Node and NPM (`sudo apt update && sudo apt install nodejs npm`)
+* Webpack (`npm install --save-dev webpack && npm install --save-dev webpack-cli`)
+* Postgres (`sudo apt update && sudo apt install postgresql postgresql-contrib` on Ubuntu)
+* Postgres user `papua/papua` (`sudo -u postgres psql -c "CREATE USER papua WITH PASSWORD 'papua';"`)
```

Comment:
There are a buttload of ways to configure Postgress....not sure we want to specify one of them - or at least we probably want to say "well, you can do this how you like, but if you don't know how you like, here's a way to do it:"

Reply:
Will generalize, offer this as "one way to do it"